src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -94,6 +106,43 @@ public void closeAngler() {
     anglerPiston.set(DoubleSolenoid.Value.kReverse);
   }
 
+  private void toSetpoint(TrapezoidProfile.State setpoint) {
```

Comment:
I think that the name `achieveState` expresses what this function does better; what about you?